.github/ISSUE_TEMPLATE/---relman-checklist.md

```diff
@@ -20,6 +20,7 @@ _Think something can be automated? Add it to [this doc](https://docs.google.com/
   - Merge to master
 - [ ] Cut LAT branch `releases/vX.X-LAT1` (this must include a-c upgrade)
 - [ ] Create GitHub pre-release build `vX.X-LAT1`([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Release-Management-Checklist#final-builds))
+- [ ] Wait for automation to email `firefox-tv@mozilla.com`, signalling that the build is done
```

Comment:
nit: do we want to put the mailing list in a public facing doc? Even if we prevent external emails from reaching the list, I'm concerned that there will be a bunch of emails to the list moderator but maybe that's disabled as well.

Do you think, "Wait for automation to email team mailing list..." is sufficient?

This applies here and in the wiki as well.